kolibri/plugins/style_guide/assets/src/app.js

```diff
@@ -1,13 +1,16 @@
 const KolibriModule = require('kolibri_module');
 const Vue = require('kolibri.lib.vue');
 const RootVue = require('./views');
+const router = require('kolibri.coreVue.router');
```

Comment:
Since we're using `router-view`, we may want to reference the vue.js router directly as `require('vue-router')`? `kolibri.coreVue.router` is actually a custom wrapper around the vue router.